lib/CodeGen/PagerandoBinning.cpp

```diff
@@ -26,17 +31,6 @@ using namespace llvm;
 #define DEBUG_TYPE "pagerando"
 
 namespace {
-struct Bin {
-  unsigned Number;
-  unsigned FreeSpace;
-
-  static const unsigned BinSize;
```

Comment:
I am a big proponent of KISS. Later we might use a smarter algorithm altogether: in the design document someone suggested using a CFG analysis to put functions that call each other in the same bin to minimize intra-bin calls.